recipes/nodejs_from_source.rb

```diff
@@ -24,9 +24,13 @@
 
 case node['platform_family']
 when 'rhel', 'fedora', 'amazon'
-  package %w(openssl-devel python3 tar)
+  package %w(openssl-devel python2 tar)
+  link '/usr/bin/python' do
+    to '/usr/bin/python2'
+    action :create
```

Comment:
The more I look at this, the more I'm concerned about doing this on a system level. This cookbook isn't managing python, it's managing nodejs. Can you elaborate on why this symlink is needed and what it's trying to accomplish?

Reply:
Working on a fix. Ark needs python in a later step.

Reply:
Is that a problem with ark? It sounds like it to me.

Reply:
ark expects python to be there.  Still working on faking it out.

Reply:
I've been meaning to add support to pyenv which would make it easier to install a system python via the package manager. Perhaps that might be helpful and then have ark depend on that?

Reply:
The issues are showing up on the centos containers. Centos usually has python2 installed for management (my memory anyway).  For me, source install on a container to make testing work is a really small edge case.  If I don't get it soon I'll block the test and move on, get it later.

Reply:
I'm very familiar with CentOS. Let me know if you want my help on figuring it out.